### DISCUSSION TITLE
mappableAs with JSON fields (Spatie translatable)

### DISCUSSION THREAD

**Discussion by barkoczi** [Jeroen-G/Explorer#243] [General]
Hello All, 

How should I implement the array in mappableAs function in case my model has title, content fields in JSON format? These fields contains the translations for different languages - I use the Spatie translatable package. 

Also the content is a JSON document generated by TIptap editor. I would like to convert it to text before send to elasticsearch database. How to do it?

Thank you for your help in advance!

**Comment by Jeroen-G** [comment]
I honestly don't know. If you can figure out how to do it for Elasticsearch in general I can tell you how to do it with this package.
